File: Programacion/Python/Tema 1/Pycharm/ejercicio1/ej2.py
def es_primo(num):
    for n in range(2, num):
        if num % n == 0:
            return "No es primo"
    return "Es primo"

n1 = int(input("Dime un número para saber si es primo o no: "))

primo = es_primo(n1)

print(f'El número que has introducido: {primo}')

File: DAM_1er_año/Programacion/Java/Repaso/src/Ej2.java
import java.util.Scanner;

public class Ej2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Dime un número para saber si es primo o no: ");
        int numero = scanner.nextInt();

        boolean primo = false;

        for (int i = 2; i < numero-1; i++) {
            if (numero % i != 0){
                primo=true;
            }else primo=false;

        }
        if (primo){
            System.out.println("Es primo");
        }else System.out.println("No es primo");

        scanner.close();
    }

}
